README.md

```diff
@@ -22,7 +22,7 @@ Make sure your .env file is up to date
 1. Navigate to the folder you want to deploy. (Example: `greenchef/services/server-greenchef`)
 2. Check out the branch you want to deploy. (Example: `release/2018-12-04`)
 3. View the `apps` constant in the `deploy.jake` file in `jakelib` to find the correct name of the docker image you want to push to
-(Example: `greenchef/services/server-greenchef` can be deployed to `consoleapi`, `web-api`, `worker`, or `scheduler`)
+(Example: `greenchef/services/server-greenchef` can be deployed to `console-api`, `web-api`, `worker`, or `scheduler`)

```

Comment:
Yeah, these changes are a little confusing with some things being `consoleapi` and some being `console-api`. If we can keep it all the same would be nice, but not a big deal.